creme/creme_core/templates/creme_core/forms/widgets/union.html

```diff
@@ -1,13 +1,11 @@
-<ul {% include "django/forms/widgets/attrs.html" %}>
+<div {% include "django/forms/widgets/attrs.html" %}>
 {% with attrs=widget.attrs %}
 {% for choice in widget.subwidget_choices %}
-  <li class="union-sub_widget">
-    <label>{# TODO: help text as tooltip ??? #}
-      <input class="union-widget-switch" type="radio" name="{{widget.name}}" value="{{choice.name}}"{% if widget.required %} required{% endif %}{% if choice.checked %} checked{% endif %}{% if widget.disabled %} disabled{% endif%}>
-      {{choice.label}}
-    </label>
-    <div class="union-sub_widget_container">{% include choice.widget.template_name with widget=choice.widget %}</div>
-  </li>
+  <div class="union-field">
+    <input class="union-field-toggle" type="radio" name="{{widget.name}}" id="union_{{choice.widget.name}}" value="{{choice.name}}"{% if widget.required %} required{% endif %}{% if choice.checked %} checked{% endif %}{% if widget.disabled %} disabled{% endif%}>
+    <label for="union_{{choice.widget.name}}">{# TODO: help text as tooltip ??? #}{{choice.label}}</label>
+    <div class="union-field-content">{% include choice.widget.template_name with widget=choice.widget %}</div>
+  </div>
 {% endfor %}
 {% endwith %}
-</ul>
\ No newline at end of file
+</div>
```

Comment:
(I had to write this comment twice; will be shorter this time...)
- I do not understand why div is better than ul/li ; ul seems meaningful.
- "field" (in "union-field") seems confusing (field & widget have a meaning in django/creme) ; "sub_widget" was OK I find. "case" could be used too (but why change?)